.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length = 88
```

Comment:
I guess I have to accept this, and I see [the operator framework](https://github.com/canonical/operator/blob/f91cf8f19d3bac03c8348b83ab6c124d021c38b2/pyproject.toml#L75) does something similar.

My sentiments remain (feel free to ignore :)) ![Old man yells at cloud](https://uploads.dailydot.com/2024/06/old-man-yells-at-cloud.jpg)